extension/content.js

```diff
@@ -128,7 +128,7 @@ function addDeleteForkLink() {
 
 	if ($postMergeContainer.length > 0) {
 		const $postMergeDescription = $postMergeContainer.find('.merge-branch-description');
-		const forkPath = $postMergeContainer.attr('data-channel').split(':')[0];
+		const forkPath = `${getUsername()}/${repoName}`;
```

Comment:
I've just found out I have a case for this: https://github.com/kurlltd/silex-doctrine-migrations-provider/pull/8

![screen shot 2016-10-24 at 23 10 46](https://cloud.githubusercontent.com/assets/506129/19662160/1f41ce78-9a3f-11e6-8aee-375562cd7fd3.png)

The actual owner of the fork is right there in the DOM with the following HTML structure:

``` html
<span class="commit-ref current-branch css-truncate user-select-contain expandable head-ref" title="clippings/silex-doctrine-migrations-provider:composer-type">
    <span class="css-truncate-target user">clippings</span>:
    <span class="css-truncate-target">composer-type</span>
</span>
```
We are wrapping that span with a link when linkifying the branch.

> Maybe we can do it from the linkified branches?

I guess that's what you mean and we can easily use it, right?

Do you need help with a reproducible case? If you want to do this and update the PR, please ping me so I can test it out as well.